templates/index.html

```diff
@@ -3,7 +3,7 @@
 <!-- wp:group {"tagName":"main","align":"full","layout":{"type":"constrained"}} -->
 <main class="wp-block-group alignfull">
 	<!-- wp:heading {"level":1,"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50"}}}} -->
-	<h1 class="wp-block-heading alignwide" style="padding-top:var(--wp--preset--spacing--50)">Posts</h1>
+	<h1 class="wp-block-heading alignwide" style="padding-top:var(--wp--preset--spacing--50)">{ __( 'Posts', 'twentytwentyfour' ) }</h1>
```

Comment:
Hi,
it is not possible to translate strings inside HTML files.

Reply:
Ok, Should I close the PR?